dragonfly/entity/state/state.go

```diff
@@ -50,6 +50,11 @@ type UsingItem struct{}
 // OnFire makes an entity show itself as on fire.
 type OnFire struct{}
 
+// Scale sets the scale of the players geometry
```

Comment:
Missing punctuation

Reply:
I don't see any issue with the punctuation on this one

Reply:
Forgot a period

Reply:
Ah, frogot documentation is supposed to have them, fixed on all files now.